src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml

```diff
@@ -0,0 +1,378 @@
+<!--
```

Comment:
Fixed in https://github.com/wso2-extensions/esb-connector-googledriverest/pull/6/